python/openassetio/hostAPI/Manager.py

```diff
@@ -557,105 +557,105 @@ def entityDisplayName(self, entityRefs, context):
 
     @debugApiCall
     @auditApiCall("Manager methods")
-    def getEntityMetadata(self, entityRefs, context):
+    def getEntityAttributes(self, entityRefs, context):
         """
-        Retrieve @ref metadata for each given entity. This may contain
-        well-known keys that you can then use to further customize your
-        handling of the entity. Some types of entity may only have
-        metadata, and no meaningful @ref primary_string.
+        Retrieve @ref attributes for each given entity. This may contain
+        well-known attributes that you can then use to further customize
+        your handling of the entity. Some types of entity may only have
+        attributes, and no meaningful @ref primary_string.
 
-        There are some well-known keys defined in the core API - @see
-        openassetio.constants. These have universally agreed meaning.
+        There are some well-known attribute names defined in the core
+        API - @see openassetio.constants. These have universally agreed
```

Comment:
[consider]/[plan] Not caused by this PR, but this doesn't render as it appears to be expected

![image](https://user-images.githubusercontent.com/7827094/144602451-781660e5-1c4d-4968-8b2d-b3db0a532d94.png)

Reply:
#128